lib/send_sms.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'Secrets'
+
+class SendSMS
+
+  def sms
+
+    require 'twilio-ruby'
```

Comment:
Normally this line would be at the top of the class definition (not inside a method).  Any thoughts on why that might be?

Reply:
Yes totally... no excuses other than in a rush and not thinking it through (along with many other violations in my code that I recognise).

Reply:
On a more positive note, it was a good call to abstract this behaviour into a separate class.